### PR TITLE
Pioneer updates

### PR DESCRIPTION
**Pull request by snwhite**

### CODE REVIEW

**Comment by snwhite** [comment]
Note: for CP02PMUI_Deploy, the components on the seafloor will have a CUID added after they are recovered.  If the WFP is confirmed lost a CUID of "UNRECOVERED" will be added.

File: deployment/CP05MOAS-GL339_Deploy.csv [hunk 1]
```diff
@@ -65,3 +65,9 @@ TS20200613,,MZ20200731,,CP05MOAS-GL339-02-FLORTM000,11,1,2020-06-13T20:09:37,202
...
 TS20200613,,MZ20200731,,CP05MOAS-GL339-05-PARADM000,11,1,2020-06-13T20:09:37,2020-07-31T18:55:48,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50238,40.08153,-70.3674,,200,200,EB
+TS20210131,,,,CP05MOAS-GL339-00-ENG000000,12,1,2021-01-31T16:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,40.09157,-71.16818,,200,200,EB
+TS20210131,,,,CP05MOAS-GL339-01-ADCPAM000,12,1,2021-01-31T16:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-50865,40.09157,-71.16818,,200,200,EB
+TS20210131,,,,CP05MOAS-GL339-02-FLORTM000,12,1,2021-01-31T16:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04472,40.09157,-71.16818,,200,200,EB
+TS20210131,,,,CP05MOAS-GL339-03-CTDGVM000,12,1,2021-01-31T16:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09338,40.09157,-71.16818,,200,200,EB
+TS20210131,,,,CP05MOAS-GL339-04-DOSTAM000,12,1,2021-01-31T16:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00161,40.09157,-71.16818,,200,200,EB
+TS20210131,,,,CP05MOAS-GL339-05-PARADM000,12,1,2021-01-31T16:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.09157,-71.16818,,200,200,EB
```
Comment (snwhite) [inline deployment/CP05MOAS-GL339_Deploy.csv:73]: Looks good.

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 2]
```diff
@@ -71,3 +71,9 @@ TS20200819,,AR48,,CP05MOAS-GL376-02-FLORTM000,12,1,2020-08-19T17:11:00,2020-11-0
...
 TS20200819,,AR48,,CP05MOAS-GL376-05-PARADM000,12,1,2020-08-19T17:11:00,2020-11-04T20:08:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50167,39.88518,-70.39578,,1000,1000,SS-1
+TS20210131,,,,CP05MOAS-GL376-00-ENG000000,13,1,2021-01-31T18:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.92755,-71.1765,,1000,1000,SS-1
+TS20210131,,,,CP05MOAS-GL376-01-ADCPAM000,13,1,2021-01-31T18:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49696,39.92755,-71.1765,,1000,1000,SS-1
+TS20210131,,,,CP05MOAS-GL376-02-FLORTM000,13,1,2021-01-31T18:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03116,39.92755,-71.1765,,1000,1000,SS-1
+TS20210131,,,,CP05MOAS-GL376-03-CTDGVM000,13,1,2021-01-31T18:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09089,39.92755,-71.1765,,1000,1000,SS-1
+TS20210131,,,,CP05MOAS-GL376-04-DOSTAM000,13,1,2021-01-31T18:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00543,39.92755,-71.1765,,1000,1000,SS-1
+TS20210131,,,,CP05MOAS-GL376-05-PARADM000,13,1,2021-01-31T18:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50167,39.92755,-71.1765,,1000,1000,SS-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL376_Deploy.csv:79]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 3]
```diff
@@ -59,3 +59,9 @@ TS20200819,,AR48,,CP05MOAS-GL379-02-FLORTM000,10,1,2020-08-19T17:18:00,2020-11-0
...
 TS20200819,,AR48,,CP05MOAS-GL379-05-PARADM000,10,1,2020-08-19T17:18:00,2020-11-04T20:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.88515,-70.39692,,1000,1000,FZ-1
+TS20210131,,,,CP05MOAS-GL379-00-ENG000000,11,1,2021-01-31T17:57:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.92648,-71.17723,,1000,1000,SS-2
+TS20210131,,,,CP05MOAS-GL379-01-ADCPAM000,11,1,2021-01-31T17:57:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49697,39.92648,-71.17723,,1000,1000,SS-2
+TS20210131,,,,CP05MOAS-GL379-02-FLORTM000,11,1,2021-01-31T17:57:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.92648,-71.17723,,1000,1000,SS-2
+TS20210131,,,,CP05MOAS-GL379-03-CTDGVM000,11,1,2021-01-31T17:57:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.92648,-71.17723,,1000,1000,SS-2
+TS20210131,,,,CP05MOAS-GL379-04-DOSTAM000,11,1,2021-01-31T17:57:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00171,39.92648,-71.17723,,1000,1000,SS-2
+TS20210131,,,,CP05MOAS-GL379-05-PARADM000,11,1,2021-01-31T17:57:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.92648,-71.17723,,1000,1000,SS-2
```
Comment (snwhite) [inline deployment/CP05MOAS-GL379_Deploy.csv:67]: Looks good.

File: deployment/CP05MOAS-GL559_Deploy.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
+TS20210131,,,,CP05MOAS-GL559-00-ENG000000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.92493,-71.1781,,1000,1000,FZ-1
+TS20210131,,,,CP05MOAS-GL559-01-ADCPAM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.92493,-71.1781,,1000,1000,FZ-1
+TS20210131,,,,CP05MOAS-GL559-02-FLORTM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.92493,-71.1781,,1000,1000,FZ-1
+TS20210131,,,,CP05MOAS-GL559-03-CTDGVM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.92493,-71.1781,,1000,1000,FZ-1
+TS20210131,,,,CP05MOAS-GL559-04-DOSTAM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00436,39.92493,-71.1781,,1000,1000,FZ-1
+TS20210131,,,,CP05MOAS-GL559-05-PARADM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92493,-71.1781,,1000,1000,FZ-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL559_Deploy.csv:7]: Looks good.

**Review by snwhite — COMMENTED** [review]
@cdobs, the times for the Glider Recoveries need to be confirmed, they are listed 2 different ways in the QLR.  As author of this PR, I can't approve.

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 5]
```diff
@@ -53,9 +53,9 @@ TS20200613,,TS20200629,,CP05MOAS-GL380-02-FLORTM000,9,1,2020-06-13T21:24:23,2020
...
\ No newline at end of file
+TS20201121,,TS20210131,,CP05MOAS-GL380-00-ENG000000,10,1,2020-11-21T17:44:00,2021-01-31T19:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.888,-70.35503,,1000,1000,FZ-1
+TS20201121,,TS20210131,,CP05MOAS-GL380-01-ADCPAM000,10,1,2020-11-21T17:44:00,2021-01-31T19:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-49016,39.888,-70.35503,,1000,1000,FZ-1
+TS20201121,,TS20210131,,CP05MOAS-GL380-02-FLORTM000,10,1,2020-11-21T17:44:00,2021-01-31T19:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04465,39.888,-70.35503,,1000,1000,FZ-1
+TS20201121,,TS20210131,,CP05MOAS-GL380-03-CTDGVM000,10,1,2020-11-21T17:44:00,2021-01-31T19:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09027,39.888,-70.35503,,1000,1000,FZ-1
+TS20201121,,TS20210131,,CP05MOAS-GL380-04-DOSTAM000,10,1,2020-11-21T17:44:00,2021-01-31T19:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00195,39.888,-70.35503,,1000,1000,FZ-1
+TS20201121,,TS20210131,,CP05MOAS-GL380-05-PARADM000,10,1,2020-11-21T17:44:00,2021-01-31T19:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50146,39.888,-70.35503,,1000,1000,FZ-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL380_Deploy.csv:61]: Looks good.

File: deployment/CP05MOAS-GL387_Deploy.csv [hunk 6]
```diff
@@ -41,9 +41,9 @@ AR34,,WJ20190617,,CP05MOAS-GL387-02-FLORTM000,7,1,2019-04-08T01:35:00,2019-06-18
...
+TS20201121,,TS20210131,,CP05MOAS-GL387-00-ENG000000,8,1,2020-11-21T17:40:00,2021-01-31T18:36:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,39.88723,-70.35692,,1000,1000,SS-1
+TS20201121,,TS20210131,,CP05MOAS-GL387-01-ADCPAM000,8,1,2020-11-21T17:40:00,2021-01-31T18:36:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,39.88723,-70.35692,,1000,1000,SS-1
+TS20201121,,TS20210131,,CP05MOAS-GL387-02-FLORTM000,8,1,2020-11-21T17:40:00,2021-01-31T18:36:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-03884,39.88723,-70.35692,,1000,1000,SS-1
+TS20201121,,TS20210131,,CP05MOAS-GL387-03-CTDGVM000,8,1,2020-11-21T17:40:00,2021-01-31T18:36:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09055,39.88723,-70.35692,,1000,1000,SS-1
+TS20201121,,TS20210131,,CP05MOAS-GL387-04-DOSTAM000,8,1,2020-11-21T17:40:00,2021-01-31T18:36:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00145,39.88723,-70.35692,,1000,1000,SS-1
+TS20201121,,TS20210131,,CP05MOAS-GL387-05-PARADM000,8,1,2020-11-21T17:40:00,2021-01-31T18:36:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50324,39.88723,-70.35692,,1000,1000,SS-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL387_Deploy.csv:49]: Looks good.

**Review by snwhite — COMMENTED** [review]
Recovery glider CSVs look good.  Glider line assignments were changed during the initial deployment.

File: deployment/CP02PMUI_Deploy.csv [hunk 7]
```diff
@@ -125,12 +125,12 @@ AR39,,AR44,,CP02PMUI-WFP01-02-DOFSTK000,14,1,2019-10-06T18:26:00,2020-06-13T16:0
...
+AR44,,,,CP02PMUI-WFP01-01-VEL3DK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-VEL3DK-00022,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-DOFSTK-02397,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-03-CTDPFK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-04-FLORTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-05-PARADK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
```
Comment (cdobs) [inline deployment/CP02PMUI_Deploy.csv:136]: Verified that stopDateTime is the same as the last transmission time in FAR 311-00055. Recovery CUID looks good for buoy as well.

Comment (rtravis-whoi) [reply deployment/CP02PMUI_Deploy.csv:136]: looks good

File: deployment/CP05MOAS-GL339_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL339_Deploy.csv:73]: Looks good

File: deployment/CP05MOAS-GL376_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL376_Deploy.csv:79]: looks good

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 3 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL379_Deploy.csv:67]: looks good

File: deployment/CP05MOAS-GL559_Deploy.csv (hunk 4 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL559_Deploy.csv:7]: looks good

File: deployment/CP05MOAS-GL380_Deploy.csv (hunk 5 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL380_Deploy.csv:61]: looks good

File: deployment/CP05MOAS-GL387_Deploy.csv (hunk 6 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL387_Deploy.csv:49]: looks good

**Review by rtravis-whoi — APPROVED** [review]

**Review by cdobs — APPROVED** [review]
2nd review completed in tandem by @snwhite and myself. everything looks good

**Pull request merged**